Przygotowanie konfiguracji do projektu

Krizzuu opened the pull request:
Przygotowanie konfiguracji Maven do projektu z domyślnym plikiem z funkcją main()

Krizzuu committed:
Init

Krizzuu committed:
Revert "Init"

This reverts commit f504f43250c37a14256350c95885e2a7a40a016d.

Krizzuu committed:
Spring Init

Krizzuu committed:
add readme.md back

HerbMat reviewed (changes requested):
You lack README.md file with description how to setup and run this project.
please add missing libraries based on this https://github.com/dorm-social-room-manager/dsrm-backend/issues/1 . And configure testcontainers library.

Krizzuu committed:
feat: added dependencies:
spring web, security, jpa data, validation, configuration processor, mariadb, testcontainers, lombok

kaczor6418 commented:
This PR is deprecated as #4 is the upp-to-date one
